virtcontainers/kata_agent.go

```diff
@@ -246,7 +246,6 @@ func (k *kataAgent) configure(h hypervisor, id, sharePath string, builtin bool,
 
 	if builtin {
 		k.proxyBuiltIn = true
-		k.state.URL, _ = k.agentURL()
```

Comment:
It's a tiny change, but it would be great if you could add a tiny unit test to `TestAgentConfigure()` :)

Reply:
Hi @jodh-intel Done